### PR TITLE
Combine add delete translations views

### PR DESCRIPTION
**Pull request by agitator**
wip - do not merge

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
         'plone.app.z3cform',
         'plone.behavior',
         'plone.dexterity',
+        'plone.api',
```
Comment (gforcada) [inline setup.py]: plone.api can not be used on core packages, as sad as it is that's the decision so far... maybe the @plone/framework-team wants to revisit this at some point :-)

**Comment by jensens** [comment]
Since this is a recommended shipped addon for core I would say it's ok. The 
difficulty is to identify which packages in core are consumers of plone.api 
and which are producers. What do others think?


Am 21. Oktober 2016 7:37:52 vorm. schrieb Gil Forcada Codinachs 
<notifications@github.com>:

**Comment by thet** [comment]
LGTM

**Comment by mauritsvanrees** [comment]
Since this has been merged, Jenkins on Plone 5.1 throws a test failure off and on:

http://jenkins.plone.org/job/plone-5.1-python-2.7-robot/1017/robot/Robot/Robot/Test%20Add%20Translation/Scenario%3A%20As%20an%20editor%20I%20can%20add%20new%20translation/

```WebDriverException: Message: arguments[0] is undefined```

Can someone have a look?
Possibly the solution is to use a newer (or older) Selenium or Firefox or whatever.

**Comment by jensens** [comment]
Ok, just looked at the robot changes, there is nothing which uses a newer Selenium of FF. Its just a bit different selectors used according to the new view. 

This more looks like a problem upstream in the overall testing. Also, IIRC, the test was green right before merge.

And, no, I have no clue what's going on.

**Comment by mauritsvanrees** [comment]
I see Gil is trying to fix it on master. I have now also tried it with a fix, and have good hope that it helps. 

The theory is: if you are using 'Wait until element is visible' to wait for an element that was not on the page at initial load, then you must first use 'Wait until page contains element', otherwise you get:

```
Message: Element not found in the cache - perhaps the page has changed since it was looked up
```

Now I look at it, that is actually a different error than I saw earlier, mentioned above... Well, let's see.

**Comment by mauritsvanrees** [comment]
Okay, that helped on Jenkins.  Well, for 5.1 anyway. 5.0 has the same error... I am rebuilding now just to see if that helps.

**Pull request merged**